src/fhir.js

```diff
@@ -50,7 +50,7 @@
         var vreadPath =  resourceHxPath.slash(":versionId || :resource.meta.versionId");
         var resourceVersionPath = resourceHxPath.slash(":versionId || :resource.meta.versionId");
 
-        var ReturnHeader = $$Header('Prefer', 'return=representation');
+        var ReturnHeader = $$Header('Prefer', cfg.headers && cfg.headers['Prefer'] ? cfg.headers['Prefer'] : 'return=representation');
```

Comment:
Tu vas pas avoir des problemes de case encore ?

Reply:
si je l'envoi avec un p minuscule oui mais après le header est censé être avec une majuscule

Reply:
J'ai ça dans la RFC de HTTP:

```
4.2 Message Headers

   HTTP header fields, which include general-header (section 4.5),
   request-header (section 5.3), response-header (section 6.2), and
   entity-header (section 7.1) fields, follow the same generic format as
   that given in Section 3.1 of RFC 822 [9]. Each header field consists
   of a name followed by a colon (":") and the field value. Field names
   are case-insensitive. The field value MAY be preceded by any amount
   of LWS, though a single SP is preferred. Header fields can be
   extended over multiple lines by preceding each extra line with at
   least one SP or HT. Applications ought to follow "common form", where
   one is known or indicated, when generating HTTP constructs, since
   there might exist some implementations that fail to accept anything
```
Donc les 2 devraient marcher, est ce qu'on en profiterait pas pour regler les problemes de casing (en mettant tout lowercase par exemple) où on voit ça plus tard ?